Modules/main.c

```diff
@@ -559,6 +559,11 @@ pymain_run_python(int *exitcode)
         goto error;
     }
 
+    // XXX Calculate runtime->sys_path_0 in getpath.py.
```

Comment:
There's a lot of XXX's in the subinterpreter changes... To me that means "there's an important thing we haven't figured out yet". If they're just "it would be nice to refactor this" or something, could you add a TODO, preferably one with an issue attached describing the work that should be done?

Reply:
Yeah, the XXX comments here are cases of "it would be nice to refactor this".  I'll drop the comments from the backport and work on tweaking them on main.